xrdsst/core/validator.py

```diff
@@ -188,7 +189,7 @@ def validate_config_client_add_or_register(ss_config, operation, errors):
     return len(errors) <= err_cnt
 
 
-def validate_config_service_desc_add_or_enable(ss_config, operation, errors):
+def validate_config_service_desc(ss_config, operation, errors):
```

Comment:
Operations are such that they are able to stand on their own and need to validate the configuration for their own only. This validation here has been widened past operation boundaries. Adding / enabling service descriptions is independent of any access definitions and their validation should not be put together with service access validation. The deeper level ``services`` configuration level is also not really required, but striclty optional, right? Lack of it should not fail validation for access definitions.

Also ``service/access`` level currently is not reflected in the error message:

```
AUTO ['service add-access']->'ss5' (redo) 
SKIPPED 'ss5':
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'access' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'services' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'access' missing required value.
```

Where the last level (if applying the required conventions in current code) should be:

```diff
-       'ss5.clients[4].service_descriptions[1]' [service add-access]: 'access' missing required value.
+      'ss5.clients[4].service_descriptions[1].services[1]' [service add-access]: 'access' missing required value.
```

The validation responsibility mixture here means that simple syntactically correct config with no access rights at all will result in having no services services added or enabled, effectively disabling these operations. This is easiest to see during autoconfiguration"

```
AUTO ['service add-description']->'ss5' (redo) 
SKIPPED 'ss5':
        'ss5.clients[4].service_descriptions[1]' [service add-description]: 'access' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-description]: 'services' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-description]: 'access' missing required value.
AUTO ['service add-access']->'ss5' (redo) 
SKIPPED 'ss5':
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'access' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'services' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service add-access]: 'access' missing required value.
AUTO ['service enable-description']->'ss5' (redo) 
SKIPPED 'ss5':
        'ss5.clients[4].service_descriptions[1]' [service enable-description]: 'access' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service enable-description]: 'services' missing required value.
        'ss5.clients[4].service_descriptions[1]' [service enable-description]: 'access' missing required value.
```

Split the validation responsibilities for access grants so that single operations stand on their own.